install/configure_administrative_access.md

```diff
@@ -10,13 +10,13 @@ BMC/controller passwords.
 ## Topics
 
 1. [Configure Keycloak account](#1-configure-keycloak-account)
-2. [Configure the Cray command line interface](#2-configure-the-cray-command-line-interface)
-3. [Set `Management` role on the BMCs of management nodes](#3-set-management-role-on-the-bmcs-of-management-nodes)
-4. [Lock management nodes](#4-lock-management-nodes)
-5. [Configure BMC and controller parameters with SCSD](#5-configure-bmc-and-controller-parameters-with-scsd)
-6. [Configure non-compute nodes with CFS](#6-configure-non-compute-nodes-with-cfs)
-7. [Upload Olympus BMC recovery firmware into TFTP server](#7-upload-olympus-bmc-recovery-firmware-into-tftp-server)
-8. [Proceed to next topic](#8-proceed-to-next-topic)
+1. [Configure the Cray command line interface](#2-configure-the-cray-command-line-interface)
+1. [Set `Management` role on the BMCs of management nodes](#3-set-management-role-on-the-bmcs-of-management-nodes)
+1. [Lock management nodes](#4-lock-management-nodes)
+1. [Configure BMC and controller parameters with SCSD](#5-configure-bmc-and-controller-parameters-with-scsd)
+1. [Configure non-compute nodes with CFS](#6-configure-non-compute-nodes-with-cfs)
+1. [Upload Olympus BMC recovery firmware into TFTP server](#7-upload-olympus-bmc-recovery-firmware-into-tftp-server)
```

Comment:
```suggestion
7. [Upload Olympus BMC recovery firmware into TFTP server](#7-upload-olympus-bmc-recovery-firmware-into-tftp-server)
```

Reply:
The style checker will complain about this.